src/helpers/get-header-paths.js

```diff
@@ -0,0 +1,9 @@
+const getHeaderLinks = (relPath) => {
+  return (relPath || '')
+    .split('/')
+    .filter(part => !part || !part.includes('.'))
```

Comment:
Do we strip out dotfiles/dotfolders? If we do not, does this filter mess up the expected behaviour?

e.g. say someone (for whatever reason) has stuff stored in `some-folder/.dotfolder/something.md`